lib/web/ui/usercontext.go

```diff
@@ -136,6 +172,7 @@ func NewUserContext(user services.User, userRoles services.RoleSet) (*UserContex
 		Users:           userAccess,
 		Tokens:          tokenAccess,
 		Nodes:           nodeAccess,
+		AccessStrategy:  requestAccess,
```

Comment:
> 		AccessStrategy:  requestAccess,

This is not part of the ACL so lets move it up and make it part of UserContext.

Reply:
revised